invenio_rdm_records/resources/serializers/csl/__init__.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""CSL JSON Serializers for Invenio RDM Records."""
+
+from __future__ import absolute_import, print_function
+
+import json
+import re
+
+from citeproc import Citation, CitationItem, CitationStylesBibliography, \
+    CitationStylesStyle, formatter
+from citeproc.source.bibtex import BibTeX
+from citeproc.source.json import CiteProcJSON
+from flask import has_request_context, request
+from webargs import fields
+from webargs.flaskparser import FlaskParser
+
+try:
+    from citeproc_styles import get_style_filepath
+    from citeproc_styles.errors import StyleNotFoundError
+except Exception:
+    import warnings
+    warnings.warn('citeproc_styles not found. '
+                  'Please install to enable Citeproc Serialization.')
+from flask_resources.serializers import MarshmallowJSONSerializer
+
+from .schema import CSLJSONSchema
+
+
+class CSLJSONSerializer(MarshmallowJSONSerializer):
+    """Marshmallow based CSL JSON serializer for records."""
+
+    def __init__(self, **options):
+        """Constructor."""
+        super().__init__(schema_cls=CSLJSONSchema, **options)
+
+    def dump_one(self, obj):
```

Comment:
major: is overwritting this function needed? maybe there's others that have the same issue... but it seems to have the same behaviour than [parent class `dump_one`](https://github.com/inveniosoftware/flask-resources/blob/master/flask_resources/serializers/json.py#L94)